alpha/action/migrate.go

```diff
@@ -5,13 +5,15 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/operator-framework/operator-registry/alpha/action/migrations"
 	"github.com/operator-framework/operator-registry/alpha/declcfg"
 	"github.com/operator-framework/operator-registry/pkg/image"
 )
 
 type Migrate struct {
 	CatalogRef string
 	OutputDir  string
+	Level      migrations.MigrationToken
```

Comment:
Thinking about this a little more, what if we put `*migrations.Migrations` here (and on the other render-based structs) instead of the level? That way:
- tests of the render-based structs could use a simple test-only `Migrations` struct to verify that the migrations are executed (it doesn't really matter _what_ those migrations are)
- Other users of the library can use other migrations if they want.